packages/web-react/src/components/TextFieldBase/TextFieldBase.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useStyleProps } from '../../hooks/styleProps';
+import { SpiritTextFieldBaseProps } from '../../types';
+import { useTextFieldBaseStyleProps } from './useTextFieldBaseStyleProps';
+
+export const TextFieldBase = (props: SpiritTextFieldBaseProps): JSX.Element => {
+  const { id, placeholder, isDisabled, isMultiline, isRequired, label, message, value } = props;
+  const { classProps, props: modifiedProps } = useTextFieldBaseStyleProps(props);
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  const ElementType: React.ElementType = isMultiline ? 'textarea' : 'input';
+
+  return (
+    <div {...styleProps} className={classNames(classProps.root, styleProps.className)}>
+      <label htmlFor={id} className={classProps.label}>
+        {label}
+      </label>
+      <ElementType
+        {...otherProps}
+        id={id}
+        className={classProps.input}
+        placeholder={placeholder}
+        disabled={isDisabled}
+        required={isRequired}
+        value={value}
```

Comment:
Yeah, you're right. Thanks for noticing! I forgot to fix it in the React implementation.

Reply:
Maybe there should be a better composition: consider using something like a `useTextFieldProps` which will return only needed props for required input. So this condition with `isMultiline` will be there. Eg.:

```js
const { value, props } = useTextFieldProps(props);

return isMultiline ? <textarea {...props}>{value}</textarea> : <input {...props} value={value} />;
```

But maybe it will be much more better to use React magic and only work with `<ElementType {...props} />`. So working with `value` as it is a transfer prop.

Meaning that when you use `<TextArea>some value</TextArea>` the value will be passed as children and when you use `<TextField value="some value" />` it will be passed as value prop.

Reply:
Ok, so if I understood well:
1) You'd stick with the `ElementType` solution? Using `ElementType` is little bit a black-box, but IMO it shouldn't be a problem. It does what it should does and the code is still readable and brief. 
2) Passing `value` in `<input>` and `<textarea>` is different. You would follow native implementation in our components as well?

Reply:
Ad 1/ Yes, it is a little bit black-box but the code will be shorter and based on the `isMultiline` prop you can switch only the `ElementType` which is IMO a simpler and more understandable way.

Ad 2/ Following the approach in step 1 we can use native behavior of both tags/components and use only `<ElementType {...props} />` in our implementation.

Therefore there will be a native usage same as `input` and `textarea` tags, eg. `<TextField value="..." />` and `<TextArea>value</TextArea>`.

But if you think that both components should be used the same way we can consider it as well.